test-workspace/vault/dendron.ref.frontmatter-tags.md

```diff
@@ -0,0 +1,10 @@
+---
+id: HyqmR36gQFN45GCI
+title: Frontmatter Tags
+desc: ''
+updated: 1627071084405
+created: 1627070098539
+tags: test
```

Comment:
can we create multiple children of this to test different cases? `fm-tags.single`, `fm-tags.multi`, ...